src/backend/graph.test.js

```diff
@@ -3,6 +3,23 @@
 import type {Address, Addressable} from "./address";
 import {sortedByAddress} from "./address";
 import {Graph} from "./graph";
+import {
```

Comment:
Personally, I'd write `import * as demoData from "./graphDemoData";` instead of this giant list, and then use `demoData.makeAddress` (etc.) below. But it's up to you.